src/basics.php

```diff
@@ -143,6 +143,15 @@ function dd($var, $showHtml = null): void
             'file' => $trace[0]['file'],
         ];
 
+        if (PHP_SAPI !== 'cli' && !empty($_SERVER['HTTP_REFERER'])) {
+            $referer = $_SERVER['HTTP_REFERER'];
+            $origin = substr($referer, 0, strlen($referer) - 1);
+            $headers = ['authorization', 'content-type', 'api-token'];
+            header('Access-Control-Allow-Credentials: true');
```

Comment:
You should also check to make sure that headers haven't already been sent. Not doing so will emit additional warnings.